src/main/java/seedu/address/logic/parser/QuestionCommandParser.java

```diff
@@ -61,6 +61,41 @@ private Index getIndex(ArgumentMultimap argMultimap) throws ParseException {
         }
     }
 
+    private AddQuestionCommand getAddQuestionCommand(Index studentIndex, ArgumentMultimap argMultimap)
+            throws ParseException {
+        assert argMultimap.getValue(PREFIX_ADD_QUESTION).isPresent();
+
+        UnsolvedQuestion question = ParserUtil.parseQuestion(argMultimap.getValue(PREFIX_ADD_QUESTION).get());
+        return new AddQuestionCommand(studentIndex, question);
+    }
+
+    private SolveQuestionCommand getSolveQuestionCommand(Index studentIndex, ArgumentMultimap argMultimap)
+            throws ParseException {
+        assert argMultimap.getValue(PREFIX_SOLVE_QUESTION).isPresent();
+
+        String userInput = argMultimap.getValue(PREFIX_SOLVE_QUESTION).get();
+
+        final Pattern solutionFormat = Pattern.compile("(?<index>[0-9]+)(\\s)(?<solution>.*)");
+        final Matcher matcher = solutionFormat.matcher(userInput.trim());
```

Comment:
I think there is a change in the command format for this part to put the solution if im not wrong. So might want to update the use format message because right now it's still the old message